package/MDAnalysis/coordinates/DCD.py

```diff
@@ -164,8 +164,14 @@ def __init__(self, filename, convert_units=True, dt=None, **kwargs):
         self.ts.dt = dt
         warnings.warn("DCDReader currently makes independent timesteps"
                       " by copying self.ts while other readers update"
-                      " self.ts inplace. This behaviour will be changed in"
-                      " 3.0 to be the same as other readers",
+                      " self.ts inplace. This behavior will be changed in"
+                      " 3.0 to be the same as other readers. In practice this"
+                      " means that you will no longer be able to store independent"
+                      " references to the Timestep object associated with the"
+                      " DCDReader. Read more at"
+                      " https://github.com/MDAnalysis/mdanalysis/issues/3899"
+                      " and https://github.com/MDAnalysis/mdanalysis/issues/3923"
+                      " to learn if this change in behavior might affect you.",
```

Comment:
```suggestion
                      " self.ts inplace. This behavior will be changed in"
                      " 3.0 to be the same as other readers. Read more at"
                      " https://github.com/MDAnalysis/mdanalysis/issues/3889"
                      " to learn if this change in behavior might affect you.",
```
For the sake of not making this take too long, could we be happy with this and we can workshop the exact details on what needs to be communicated there?

Reply:
sounds good to me